content/blog/gain-platform-superpowers-with-the-kebap-stack/index.md

```diff
@@ -0,0 +1,704 @@
+  linkedin: |
+    See how well Pulumi plays with other CNCF projects in a Cloud Native world. Learn how I combined following projects to create a standardized GitOps stack:
+    - Kubernetes
+    - External Secrets Operator
+    - Backstage
+    - Argo CD
+    - Pulumi
+
+    Learn how it can help you streamline your software delivery process by using standard tools and practices across your infrastructure projects. 
+
+    This helps reduce the time required to establish development and production environments from months to minutes. #GitOps #Kubernetes #ArgoCD #Pulumi #Backstage
+---
+
+In today's complex digital landscape, organizations are increasingly turning to platform engineering to optimise their
+software delivery processes and maximize efficiency. The growing complexity of modern applications, coupled with the
+need for rapid, secure, and scalable deployments, has created a pressing demand for robust Internal Developer
+Platforms (IDPs).
+
+IDPs are a key component of modern platform engineering strategies. An IDP is a self-service layer that sits on top of
+an organization's infrastructure, abstracting away complexity and providing developers with the tools and environments
+they need to build, test, and deploy applications efficiently.
+
+Key features of an effective IDP include:
+
+1. Self-service capabilities: Developers can provision resources, environments, and services without relying on ops
+   teams.
+2. Standardization: IDPs enforce best practices and compliance standards across the organization.
+3. Automation: Repetitive tasks are automated, reducing manual errors and freeing up developer time.
+4. Observability: Built-in monitoring and logging provide insights into application and infrastructure performance.
+5. Integration: IDPs seamlessly integrate with existing tools and workflows in the development lifecycle.
+
+By implementing an IDP, organizations can significantly reduce cognitive load on developers, accelerate time-to-market,
+improve security and compliance, and foster a culture of innovation. IDPs act as a force multiplier, allowing
+development teams to focus on creating business value while the platform handles the underlying complexity of
+cloud-native infrastructure.
+
+`KEBAP`, as a reference architecture, demonstrates how to construct an IDP using best-in-class CNCF technologies,
+orchestrated and managed through Pulumi's infrastructure as code capabilities.
+
+Platform engineering addresses these challenges by creating a layer of abstraction between development teams and the
+underlying infrastructure. This approach aims to simplify workflows, standardize practices, and empower developers to
+focus on creating business value rather than grappling with intricate infrastructure details.
+
+The [Cloud Native Computing Foundation](https://www.cncf.io/) (CNCF) landscape offers a rich ecosystem of tools and
```

Comment:
I think we got the causation flipped here. A lot of people are using a fully CNCF stack or at least using K8s. They want to make this more self-service for their developers through a platform. There are pieces that can go into it from the CNCF stack, but how do you orchestrate it all together and fit it together is the question that is overwhelming